README.md

```diff
@@ -1,49 +1,71 @@
+			- [**CaptureWritePCLVis3D**](https://github.com/zivid/cpp-extra-samples/blob/master/Applications/Basic/Visualization/CaptureWritePCLVis3D/CaptureWritePCLVis3D.cpp) - This example shows how capture a Zivid point cloud, save it to a .PCD file format, and visualize it.
+			- [**ReadPCLVis3D**](https://github.com/zivid/cpp-extra-samples/blob/master/Applications/Basic/Visualization/ReadPCLVis3D/ReadPCLVis3D.cpp) - This example shows how to read a PCL point cloud and visualize it.
+		- [**FileFormats**](https://github.com/zivid/cpp-extra-samples/tree/master/Applications/Basic/FileFormats)
+			- [**ReadZDF**](https://github.com/zivid/cpp-extra-samples/tree/master/Applications/Basic/FileFormats/ReadZDF/ReadZDF.cpp) - This example shows how to import and display a Zivid point cloud from a .ZDF file.
+	- [**Advanced**](https://github.com/zivid/cpp-extra-samples/tree/master/Applications/Advanced)
+		- [**HandEyeCalibration**](https://github.com/zivid/cpp-extra-samples/tree/master/Applications/Advanced/HandEyeCalibration)
+			- [**HandEyeCalibration**](https://github.com/zivid/cpp-extra-samples/tree/master/Applications/Advanced/HandEyeCalibration/HandEyeCalibration/HandEyeCalibration.cpp)
+			- [**UtilizeEyeInHandCalibration**](https://github.com/zivid/cpp-extra-samples/blob/master/Applications/Advanced/HandEyeCalibration/UtilizeEyeInHandCalibration/UtilizeEyeInHandCalibration.cpp) - Transform a 3D point from camera frame to robot base frame using hand-eye calibration matrix.
+			- [**PoseConversions**](https://github.com/zivid/cpp-extra-samples/blob/master/Applications/Advanced/HandEyeCalibration/PoseConversions/PoseConversions.cpp):
+				- Convert to/from Transformation Matrix (Rotation Matrix + Translation Vector)
+		- [**Downsample**](https://github.com/zivid/cpp-extra-samples/tree/master/Applications/Advanced/Downsample)  - This example shows how to import a Zivid point cloud from a .ZDF file and downsample it.
+			- **Dependencies:**
+				- [Eigen](http://eigen.tuxfamily.org/) version 3.3.7 or newer
+		- [**CaptureUndistortRGB**](https://github.com/zivid/cpp-extra-samples/blob/master/Applications/Advanced/CaptureUndistortRGB/CaptureUndistortRGB.cpp) - Use Zivid camera intrinsics to undistort an RGB image. This example will prompt the user for whether to capture a 2D or a 3D image. In both instances it will operate on a 2D image. However, in the 3D case it will extract 2D image from a ZDF point cloud. The 2D variant is faster.
+			- **Dependencies:**
+				- [OpenCV](https://opencv.org/) version 4.0.1 or newer
+		- [**CreateDepthMap**](https://github.com/zivid/cpp-extra-samples/blob/master/Applications/Advanced/CreateDepthMap/CreateDepthMap.cpp) - Import a ZDF point cloud and convert it to OpenCV format, then extract and visualize depth map.
+			- **Dependencies:**
+				- [OpenCV](https://opencv.org/) version 4.0.1 or newer
+
+## Samples to be removed
```

Comment:
I don't understand this heading...

Reply:
It's for your eyes only :) I guess the fact that these are deleted will appear in a git commit